gRPC LROs need to support empty return types

This is blocking the latest update to storage control. See: 

https://pantheon.corp.google.com/cloud-build/builds;region=us-central1/4216c827-9633-4bc3-a5bc-2b3a313bf8eb?project=rust-sdk-testing

<details>
<summary>Build error: </summary>

```
error[E0433]: failed to resolve: could not find `protobuf` in `google`
    --> src/storage/src/generated/gapic_control/transport.rs:2158:38
     |
2158 |             .to_msg::<crate::google::protobuf::Empty>()
     |                                      ^^^^^^^^ could not find `protobuf` in `google`
   Compiling google-cloud-workflows-v1 v1.6.0 (/workspace/src/generated/cloud/workflows/v1)
   Compiling google-cloud-speech-v2 v1.6.0 (/workspace/src/generated/cloud/speech/v2)
   Compiling google-cloud-pubsub v0.32.3-preview (/workspace/src/pubsub)
   Compiling google-cloud-showcase-v1beta1 v1.0.0 (/workspace/src/generated/showcase)
   Compiling google-cloud-bigquery-v2 v1.0.0 (/workspace/src/generated/cloud/bigquery/v2)
error[E0599]: no method named `to_proto` found for struct `google_cloud_wkt::Empty` in the current scope
    --> src/storage/src/generated/gapic_control/transport.rs:2104:14
     |
2101 |           "type.googleapis.com/google.protobuf.Empty" => value
     |  ________________________________________________________-
2102 | |             .to_msg::<wkt::Empty>()
2103 | |             .map_err(ConvertError::other)?
2104 | |             .to_proto()
     | |             -^^^^^^^^ method not found in `google_cloud_wkt::Empty`
     | |_____________|
     |
```
</details>

--- a/src/storage/src/lib.rs
+++ b/src/storage/src/lib.rs
@@ -151,4 +151,8 @@ pub(crate) mod google {
             }
         }
     }
+    #[allow(unused_imports)]
+    pub mod protobuf {
+        pub use gaxi::prost::Empty;
+    }
 }

